src/views/DogCard/DogList.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import './DogList.css';
+
+import { fetchDogs } from '../../services/fetchdogs';
+
+export default function Home() {
+  const [dog, setDog] = useState([]);
+
+  useEffect(()=> {
+    const fetchDoggosFunct = async () => {
+      const data = await fetchDogs();
+      setDog(data);
+    };
+    fetchDoggosFunct();
+  }, []);
+
+  return (
+    <div className='doggo'>
+      {dog.map((data) => (
+        <div className='dogpic' key={data.id}>
+          <Link to={`/dogs/${data.id}`}>
+            <img src={data.image} />
+          </Link>
+
+          <div className='dogname'>
+            {data.name}
+          </div>
```

Comment:
We may want to wrap the link around the image and name. I was unable to click a link to my dog detail page unless an image was added. Another way around this would be to add 'required' to the input field for the image link so an image must be uploaded.